src/gss_sec_ctx.c

```diff
@@ -1263,6 +1263,34 @@ static uint32_t gssntlm_sasl_ssf(uint32_t *minor_status,
     return GSSERRS(retmin, retmaj);
 }
 
+/* GSS_C_INQ_SSPI_SESSION_KEY 1.2.840.113554.1.2.2.5.5 */
+static const gss_OID_desc sspi_session_key_oid = {
+  11, discard_const("\x2a\x86\x48\x86\xf7\x12\x01\x02\x02\x05\x05")
+};
```

Comment:
This OID is already available in gssapi_ext.h as GSS_C_INQ_SSPI_SESSION_KEY so we do not need to redeclare it here.